### PR TITLE
Add set host key to jump host context manager

### PR DESCRIPTION
**Pull request by tybruno**

### CODE REVIEW

File: paramiko_jump/client.py [hunk 1]
```diff
@@ -102,35 +103,34 @@ def _auth(
...
-            gss_auth=False,
-            gss_kex=False,
-            gss_deleg_creds=True,
-            gss_host=None,
-            banner_timeout=None,
-            auth_timeout=None,
-            gss_trust_dns=True,
-            passphrase=None,
-            disabled_algorithms=None,
+        self,
```
Comment (andrewschenck) [inline paramiko_jump/client.py:106]: Please don't do this -- it's breaking pylint. These are intentionally formatted the way they were. Is this black doing this?

File: paramiko_jump/client.py [hunk 2]
```diff
@@ -102,35 +103,34 @@ def _auth(
...
     ):  # pylint: disable=R0913,R0914
         if self._jump_session is not None:
             if sock is not None:
-                raise ValueError('jump_session= and sock= are mutually '
-                                 'exclusive')
+                raise ValueError("jump_session= and sock= are mutually " "exclusive")
             transport = self._jump_session._transport  # pylint: disable=W0212
             sock = transport.open_channel(
-                kind='direct-tcpip',
+                kind="direct-tcpip",
```
Comment (andrewschenck) [inline paramiko_jump/client.py:133]: Please don't do this either

File: paramiko_jump/client.py [hunk 3]
```diff
@@ -23,7 +23,7 @@
 )
 
 from paramiko.client import SSH_PORT, SSHClient
-
+from paramiko import AutoAddPolicy
```
Comment (andrewschenck) [inline paramiko_jump/client.py:26]: this shuold be imported before our own stuff

File: paramiko_jump/client.py [hunk 4]
```diff
@@ -102,35 +103,34 @@ def _auth(
...
+        auth_timeout=None,
+        gss_trust_dns=True,
+        passphrase=None,
+        disabled_algorithms=None,
     ):  # pylint: disable=R0913,R0914
         if self._jump_session is not None:
             if sock is not None:
-                raise ValueError('jump_session= and sock= are mutually '
-                                 'exclusive')
+                raise ValueError("jump_session= and sock= are mutually " "exclusive")
```
Comment (andrewschenck) [inline paramiko_jump/client.py:130]: I really prefer single quotes, but most of all I prefer to not have alternating styles in a single project.

**Comment by andrewschenck** [comment]
line 26 and line 232/233, 238/239 are the only changes I think should be in this diff -- the rest appears to be a code formatter tool

File: paramiko_jump/client.py (hunk 1 above)
Comment (tybruno) [reply paramiko_jump/client.py:106]: Hmm black must have messed it up. I ran black before pushing.

File: paramiko_jump/client.py (hunk 2 above)
Comment (tybruno) [reply paramiko_jump/client.py:133]: Black must be changing the quotes because I didn't change them.

File: paramiko_jump/client.py (hunk 3 above)
Comment (tybruno) [reply paramiko_jump/client.py:26]: That makes sense!

**Comment by tybruno** [comment]
Closed because I created a new branch that only changes the piece that needed to change and did not run black

**Pull request closed without merging**